interchain/interchain-end_to_end_testing/src/bin/ibc_hook.rs

```diff
@@ -68,11 +68,14 @@ pub fn test_ibc_hook() -> AnyResult<()> {
         )?
         .interchain_channel;
 
-    let (abstr_juno, abstr_juno2) = abstract_starship_interfaces(
-        &interchain,
-        &juno_abstract_deployer,
-        &juno2_abstract_deployer,
-    )?;
+    // let (abstr_juno, abstr_juno2) = abstract_starship_interfaces(
```

Comment:
Revert please, I forgot to do it myself

Reply:
Only this or lines below as well? Does await works as expected now on >2 chains?

Reply:
Yeah it works perfectly here, there shouldn't be any problem (that I know of)